flocker/node/agents/blockdevice.py

```diff
@@ -869,12 +878,12 @@ def run(self, deployer):
         )
         device = api.get_device_path(volume.blockdevice_id)
 
-        # This duplicates CreateFilesystem now.
-        check_output(["mkfs", "-t", "ext4", device.path])
+        create = CreateFilesystem(volume=volume, filesystem=u"ext4")
```

Comment:
I still think the logging assertion change is fine: all it does is not blow up if some other code happens to log a related action. Which it does, legitimately, in this case.